### PR TITLE
add affect details in tracker validations

### PR DESCRIPTION
**Pull request by costaconrado**
This PR adds affects details in tracker validations related to affects.

Closes OSIDB-3439.

### CODE REVIEW

**Review by JakubFrejlach — CHANGES_REQUESTED** [review]
Couple of things:
* `_validate_tracker_duplicate` should be handled the same way
* there should be probably a migration to convert these changed alerts to a new description format

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [4.3.2] - 2024-09-19
 ### Changed
 - Update the release documentation (OSIDB-3384)
+- Tracker validations show affect's module/component (OSIDB-3439)
```
Comment (jobselko) [inline docs/CHANGELOG.md:45]: Wrong ## section

Comment (costaconrado) [reply docs/CHANGELOG.md:45]: nice catch, I didn't paid enough attention in the rebase.

**Review by jobselko — APPROVED** [review]
I am not familiar with the PR, but changelog LGTM :)

**Review by osoukup — APPROVED** [review]
LGTM

**Review by JakubFrejlach — APPROVED** [review]
LGTM now!

File: docs/CHANGELOG.md
Comment (jobselko) [inline docs/CHANGELOG.md:1]: Please rebase on top of master

**Review by jobselko — CHANGES_REQUESTED** [review]
Could you please pull the latest master and rebase on top of that?

**Review by jobselko — APPROVED** [review]
Changelog LGTM now :)

**Pull request merged**